examples/text_to_image/train_text_to_image_lora.py

```diff
@@ -903,18 +918,19 @@ def collate_fn(examples):
 
     if accelerator.is_main_process:
         for tracker in accelerator.trackers:
-            if tracker.name == "tensorboard":
-                np_images = np.stack([np.asarray(img) for img in images])
-                tracker.writer.add_images("test", np_images, epoch, dataformats="NHWC")
-            if tracker.name == "wandb":
-                tracker.log(
-                    {
-                        "test": [
-                            wandb.Image(image, caption=f"{i}: {args.validation_prompt}")
-                            for i, image in enumerate(images)
-                        ]
-                    }
-                )
+            if len(images) != 0:
+                if tracker.name == "tensorboard":
+                    np_images = np.stack([np.asarray(img) for img in images])
+                    tracker.writer.add_images("test", np_images, epoch, dataformats="NHWC")
+                if tracker.name == "wandb":
+                    tracker.log(
+                        {
+                            "test": [
+                                wandb.Image(image, caption=f"{i}: {args.validation_prompt}")
+                                for i, image in enumerate(images)
+                            ]
+                        }
+                    )
```

Comment:
added length check for when `num_validation_images` is zero, this throws an error trying to stack an empty array.
I had to set `num_validation_images` to zero as the dummy pipeline throws an error during inference. Would be ideal to fix the dummy pipeline with the training script, but this is an ok workaround for now